### ISSUE TITLE
Feature Request: Give users a lecture when they use gsudo.

### ISSUE THREAD

**Issue by septsea** [gerardog/gsudo#332]
## Description

One is supposed to use `gsudo` (or `sudo`) with care, so we are convinced that it is good to give users a lecture when they use gsudo. By "giving a lecture", I mean printing the following text.
```plaintext
We trust you have received the usual lecture from the local System
Administrator. It usually boils down to these three things:

    #1) Respect the privacy of others.
    #2) Think before you type.
    #3) With great power comes great responsibility.
```

## Proposed technical details

Printing the text above will work.

It will be better if users can determine whether the lecture is always shown or is shown just once.

**Comment by gerardog** [comment]
I obviously love and respect Unix/Linux sudo, but why this should be included in `gsudo`? Please convince me.

Also, should it keep the original text? Shouldn't it instead write an updated security warning, circa 2024?

As the author and maintainer... am I allowed to plug some text of my own in there? For example: `Donate! give me money!` for example. Ahh I already see you frown!

**Comment by septsea** [comment]
> Also, should it keep the original text? Shouldn't it instead write an updated security warning, circa 2024?

Not necessarily; I guess that any warning that gives users a lecture is fine.

---

> As the author and maintainer... am I allowed to plug some text of my own in there? For example: `Donate! give me money!` for example. Ahh I already see you frown!

I guess that most users will dislike it, but I do not really care. (I did not frown; instead I laughed a bit.)

(I really wanted to give you money, but "Donations to this recipient aren't supported in this country".)

---

> I obviously love and respect Unix/Linux sudo, but why this should be included in `gsudo`? Please convince me.

There is no harm in giving users a lecture at least for the first time. No one says that there has to be a warning, but including this is better. I am just giving advice.

**Comment by gerardog** [comment]
The lecture coul also improve the scenario described in #221.

**Comment by septsea** [comment]
You are right. Most users do not disable UAC themselves, but some do and still need some kinds of warnings. I did not think about such a scenario.

**Comment by mattcargile** [comment]
No need for a lecture as the users have to reach out and install this on their own. I would expect maybe Microsoft Windows to have their lecture in the OS.
